lib/Storage/AmazonS3.php

```diff
@@ -404,7 +404,7 @@ public function touch($path, $mtime = null) {
 			$mtime = time();
 		}
 		$metadata = [
-			'lastmodified' => gmdate(\Aws\Common\Enum\DateFormat::RFC1123, $mtime)
+			'lastmodified' => gmdate(\DateTime::RFC1123, $mtime)
```

Comment:
please rebase on top of https://github.com/owncloud/files_external_s3/pull/2/commits